### PR TITLE
Fixed Get-AzVpn*Certificate cmdlets

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRevokedCertificateCommand.cs [hunk 1]
```diff
@@ -50,13 +50,17 @@ public override void Execute()
         {
             base.Execute();
             var vnetGateway = this.GetVirtualNetworkGateway(this.ResourceGroupName, this.VirtualNetworkGatewayName);
+            if (vnetGateway.VpnClientConfiguration == null)
+            {
+                throw new ArgumentException(string.Format("No revoked certificates found on VirtualNetworkGateway {0}", VirtualNetworkGatewayName));
```
Comment (maddieclayton) [inline src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRevokedCertificateCommand.cs]: Any string that the user sees must go into Resources.resx

File: src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRootCertificateCommand.cs [hunk 2]
```diff
@@ -50,13 +50,17 @@ public override void Execute()
         {
             base.Execute();
             var vnetGateway = this.GetVirtualNetworkGateway(this.ResourceGroupName, this.VirtualNetworkGatewayName);
+            if (vnetGateway.VpnClientConfiguration == null)
+            {
+                throw new ArgumentException(string.Format("No root certificates found on VirtualNetworkGateway {0}", VirtualNetworkGatewayName));
```
Comment (maddieclayton) [inline src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRootCertificateCommand.cs]: same comment here

**Comment by EvgenyAgafonchikov** [comment]
Adding DoNotMerge for issues fixes PRs until team's decision on target branch (master or Network monthly)

File: src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRevokedCertificateCommand.cs (hunk 1 above)
Comment (maddieclayton) [reply src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRevokedCertificateCommand.cs]: Ping - please put any string that a user will see into Resources.resx

File: src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRootCertificateCommand.cs (hunk 2 above)
Comment (maddieclayton) [reply src/Network/Network/VirtualNetworkGateway/GetAzureVpnClientRootCertificateCommand.cs]: Ping

**Pull request merged**